packages/rspack-plugin-preact-refresh/tests/hotCases/hook/useContext#provide/__snapshots__/web/1.snap.txt

```diff
@@ -37,10 +37,11 @@ __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   App: function() { return App; }
 });
-/* harmony import */var react_jsx_dev_runtime__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! react/jsx-dev-runtime */ "../../../../../../node_modules/preact/compat/jsx-dev-runtime.js");
-/* harmony import */var preact_compat__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! preact/compat */ "../../../../../../node_modules/preact/compat/dist/compat.js");
+/* harmony import */var react_jsx_dev_runtime__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! react/jsx-dev-runtime */ "../../../../../../node_modules/.pnpm/preact@10.22.0/node_modules/preact/compat/jsx-dev-runtime.js");
```

Comment:
snapshot contains .pnpm path seems not ideal

Reply:
> We can replace it with a placeholder, like Rsbuild [web-infra-dev/rsbuild@`main`/scripts/test-helper/src/pathSerializer.ts#L54-L58](https://github.com/web-infra-dev/rsbuild/blob/main/scripts/test-helper/src/pathSerializer.ts?rgh-link-date=2024-07-15T08%3A49%3A27Z#L54-L58)

yeah, I know that, but there is a problem that the bundle size is still a little unstable because of the node_modules change when updating dependencies

<img width="300" alt="image" src="https://github.com/user-attachments/assets/8980b527-f465-41ba-b377-8d2d45740eb1">

If we replace the chunkId, the bundle size calculation maybe a little different

Reply:
So the snapshot is already unstable because of the bundle size. I think it is ok not to replace the pnpm path.

Reply:
> I think it is ok not to replace the pnpm path.

I agree with this suggestion. At present, there are almost 20 snapshot test files being affected by pnpm.

10 rspack-plugin-preact-refresh (It will be moved to other repo soon)
7 hash (can't be dealed with by replacing pnpm path)
3 statsTest (with size)

Reply:
@hardfist cc